nextflow.config

```
@@ -25,16 +169,16 @@ params {
 
     // Boilerplate options
     outdir                     = './results'
```

The template suggests here "null"
```suggestion
    outdir                     = null
```

---

To elaborate: When running the pipelines in the cloud it can happen that with default (i.e. outdir = ./results) the results folder is on a remote and later auto-deleted instance and therefore never can be obtained. Thats why outdir is now required to be specified instead of having a default.